e2e-tests/gef/cypress/e2e/facility/pre-submission/facility-end-date-enabled/about-facility.spec.js

```diff
@@ -174,12 +167,11 @@ context('About Facility Page  - feature flag enabled', () => {
       cy.visit(relative(`/gef/application-details/${application.id}/facilities/${facilityId}/about-facility`));
       cy.keyboardInput(aboutFacility.facilityName(), 'Name');
       aboutFacility.shouldCoverStartOnSubmissionNo().click();
-      cy.keyboardInput(aboutFacility.coverStartDateDay(), dateConstants.todayDay);
-      cy.keyboardInput(aboutFacility.coverStartDateMonth(), dateConstants.todayMonth);
-      cy.keyboardInput(aboutFacility.coverStartDateYear(), dateConstants.todayYear);
-      cy.keyboardInput(aboutFacility.coverEndDateDay(), dateConstants.twoDaysDay);
-      cy.keyboardInput(aboutFacility.coverEndDateMonth(), dateConstants.twoDaysMonth);
-      cy.keyboardInput(aboutFacility.coverEndDateYear(), dateConstants.twoDaysYear);
+
+      cy.completeDateFormFields({ idPrefix: 'cover-start-date' });
+
+      cy.completeDateFormFields({ idPrefix: 'cover-end-date', date: twoDaysAgo });
```

Comment:
This was previously two days in the future (It needs to be after the cover start date, I think using `tomorrow` would also be fine).

Reply:
Thanks @BethThomas141  have updated to use `twoDays` to keep it the same as before